Dockerfile

```diff
@@ -1,16 +1,21 @@
-FROM python:3-buster
+FROM amazon/aws-lambda-python:3
 
-ARG MANAGER_VERSION="2.1.1"
+ARG MANAGER_VERSION="2.1.2"
 
 # Install build dependencies
 RUN pip3 install --no-cache-dir --disable-pip-version-check --progress-bar off 'poetry==1.1.4'
 
 # Install ecs manager
-WORKDIR /home/
-RUN curl --silent --show-error --location https://github.com/trussworks/terraform-aws-lambda-ecs-manager/archive/"$MANAGER_VERSION".zip > manager.zip \
- && unzip manager.zip
-
 WORKDIR /home/terraform-aws-lambda-ecs-manager-"$MANAGER_VERSION"
+
+COPY README.md ./
+COPY mypy.ini ./
+COPY poetry.lock ./
+COPY pyproject.toml ./
+COPY pytest.ini ./
+COPY setup.cfg ./
+COPY functions ./functions
+
```

Comment:
I believe `mypy.ini` and `pytest.ini` are dev dependencies and don't need to be in the container. This sure beats `COPY . .` regardless though.